source/stylesheets/refills/_hero.scss

```diff
@@ -27,34 +27,29 @@
     margin: auto;
     padding: 3.5em;
     text-align: center;
+  }
+
+  .hero-copy {
+    text-align: center;
+  }
 
-    .hero-copy {
-      text-align: center;
-      
-      h1 {
-        font-size: 1.6em;
-        margin-bottom: 0.5em;
-
-        @include media($large-screen) {
-          font-size: 1.8em;
-        }
-      }
-
-      p {
-        font-weight: 200;
-        line-height: 1.4em;
-        margin: 0 auto 3em auto; 
-
-        @include media($large-screen) {
-          font-size: 1.1em;
-          max-width: 40%;
-        }
-      }
+  h1 {
+    font-size: 1.6em;
+    margin-bottom: 0.5em;
+
+    @include media($large-screen) {
+      font-size: 1.8em;
     }
+  }
+
+  p {
+    font-weight: 200;
+    line-height: 1.4em;
+    margin: 0 auto 3em auto; 
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 auto 3em` instead of `0 auto 3em auto`